src/simulator/vector_map.cpp

```diff
@@ -370,16 +370,18 @@ void VectorMap::Cleanup() {
   const float kMinLineLength = 0.05;
   vector<Line2f> new_lines;
   for (size_t i = 0; i < lines.size(); ++i) {
-    const Line2f l1 = lines[i];
+    const Line2f& l1 = lines[i];
```

Comment:
why do you need to make this a reference?

Reply:
That was a typo on my part. Fixed!